app/controllers/transactions.js

```diff
@@ -35,6 +35,19 @@ module.exports = function(app) {
     var group = args.group || {};
     var paymentMethod = args.paymentMethod || {};
 
+    if (transaction.amount > 0 && transaction.txnCurrencyFxRate) {
+      // populate netAmountInGroupCurrency for donations
+        transaction.netAmountInGroupCurrency =
+          Math.round((transaction.amountInTxnCurrency
+            - transaction.platformFeeInTxnCurrency
+            - transaction.hostFeeInTxnCurrency
+            - transaction.paymentProcessorFeeInTxnCurrency)
+          *transaction.txnCurrencyFxRate);
+    } else {
+      // populate netAmountInGroupCurrency for "Add Funds" and Expenses
+      transaction.netAmountInGroupCurrency = transaction.amount*100;
```

Comment:
Since the host is likely in the same currency than the collective, it feels like we should first convert the transaction.amount to the right currency, and then apply the host fees. I also wonder if the paymentProcessor takes their fees after or before the FXRate.